### DISCUSSION TITLE
Ports/URL used for Firewall rules

### DISCUSSION THREAD

**Discussion by Maffycake** [AykutCevik/Geolocate-IP-Browser-Extension#24] [Q&A] [answered]
Hi All Geolocate users and Aykut,

Firstly, I want to say what a fantastic app this is. Simple, but works well.

I've noticed recently that my network firewall is blocking the app from running when in the office. Watching the logs, i've narrowed down a few of the IP's that appear to be what should be allowed from my system for useful diagnostic purposes.

Assume it needs to be able to contact a particular server or group of servers. Could i get a confirmation whether there's a paticular IP or URL I need to set an exclusion for in the firewall to allow this traffic through?? I don't want to add more IP's or allowances than necessary.

Thanks,
Matt

**Comment by AykutCevik** [comment, marked as answer]
Hello @Maffycake,

I guess it is because of the ports. The extension needs to be able to open https connections through the ports `8443` and `9443`.  Can you check this?
Other than that the hostnames `ipv4.aykutcevik.com` and `ipv6.aykutcevik.com` are used to make the request.

**Comment by Maffycake** [comment]
Hi Aykut,

Thanks for your response. This is perfect. It worked for me. I've put a rule allowing this traffic to that URL.

Regards,
Matt
